doc/API.md

```diff
@@ -80,8 +80,10 @@ The `experimental`, `ACT` and `section508` tags are only added to some rules. Ea
 | ---------------- | ---------------------------------------------------- |
 | `wcag2a`         | WCAG 2.0 Level A                                     |
 | `wcag2aa`        | WCAG 2.0 Level AA                                    |
+| `wcag2aaa`       | WCAG 2.0 Level AAA                                   |
 | `wcag21a`        | WCAG 2.1 Level A                                     |
 | `wcag21aa`       | WCAG 2.1 Level AA                                    |
+| `wcag21aaa`      | WCAG 2.1 Level AAA                                   |
```

Comment:
We don't have any wcag21aaa rules. Probably shouldn't list it as a valid tag until we have at least one as trying to use it will fail:

```js
await axe.run({ runOnly: ['wcag21aaa']}) //Could not find tags `wcag21aaa`
```